pkg/testers/ginkgo/ginkgo.go

```diff
@@ -70,6 +70,20 @@ func (t *Tester) Test() error {
 
 func (t *Tester) pretestSetup() error {
 	if config := os.Getenv("KUBECONFIG"); config != "" {
+		// The ginkgo tester errors out if the kubeconfig provided
```

Comment:
now that artifacts is relative and all deployers expose absolute paths for their generated kubeconfigs this section is probably no longer needed ?

Reply:
FYI i also suspect this bug is most likely because of passing the kubeconfig path directly as strings (line 55) which could potentially contain shell specific characters like `~` for `$HOME`
we can probably execute it as `sh -c` and drop the equals while assigning flag values, just separated args should also work.

Reply:
Yeah, based on current code paths it isn't necessary in the tester itself any more, but we did talk about enforcing it at more boundaries. I'm happy to go either way with it.

As for the bug, I don't think its shell-specific chars (could be wrong) because I remember seeing in the test output a pure relative path `_artifacts/kubetest2-kubeconfig` being passed to ginkgo and it failing.

Reply:
okay let's drop the tester changes if the artifacts change was sufficient for the gce deployer.
we can consider adding it back later if we run into issues again.

Reply:
Actually, looking at it again. The default dir will work fine (enforced abs) but if the user passes a relative path for `--artifacts` and also uses the ginkgo tester (thus not obeying the updated usage message) kt2 will not error until after Up(), when it calls the tester. The error will also be somewhat obscure to someone who hasn't gone through this issue.

To fix it I think we either enforce it in the ginkgo tester (as we do in the current code) or in the deployer, which is, in my opinion, inappropriate because it only matters to the tester, not the deployer.

Reply:
I guess we could still validate and expand user passed flag value.
but let's go forward with this current approach since it's fixing a broken behavior.

note: that eventually we'll be having cases where the kubeconfigpath is actually a comma separated list.